release_dashboard/lib/services/local_conductor.dart

```diff
@@ -90,4 +101,12 @@ class LocalConductorService extends ConductorService {
     _engineCheckoutDirectory = await startContext.engine.checkoutDirectory;
     _frameworkCheckoutDirectory = await startContext.framework.checkoutDirectory;
   }
+
+  /// Sync the release status after cleaning the state file.
```

Comment:
This is more of an implementation detail than a summary. This is probably a case where you don't need to discuss what conductor_core is doing, and just rely on the parent's dartdocs

Reply:
The parent's doc doesn't specify synching the status after the clean. That's what I wanted to emphasize.

Reply:
I added the parent's doc here.

Reply:
No need to duplicate, they're inherited in the VSCode IDE

Reply:
ok removed the comment.